javascript/javascript1/week3/series_life.js

```diff
@@ -0,0 +1,47 @@
+const seriesDurations = [
+    {
+      title: 'Game of thrones',
+      days: 3,
+      hours: 1,
+      minutes: 0,  
+    },
+    {
+      title: 'Sopranos',
+      days: 3,
+      hours: 14,
+      minutes: 0,
+    },
+    {
+      title: 'The Wire',
+      days: 2,
+      hours: 12,
+      minutes: 0,
+    }
+  ];
+  
+  const myFavoriteSeries={
+      title: "Two and half men",
+      days: 5,
+      hours: 11,
+      minutes:0,
+  };
+  seriesDurations.push(myFavoriteSeries);// my favorite series added
+   
+ 
+  // calculating: for each series time spent percentage of an average lifespan of 80 years
+  const lifespanInHours=700800;//All are in days and hours so all durations are calculated in to hours
+  
+  console.log(seriesDurations[0].title+" took "+((seriesDurations[0].days*24+seriesDurations[0].hours)
```

Comment:
ok here you're clearly repeating the same calculation in many places, so it's a perfect place to make a function.

Reply:
Also, what about minute? Of course in your data set, `minutes` is always 0, but in "real life" that might change.